### PR TITLE
Breaking: Rename analytical_statement to analytical_statements

### PR DESCRIPTION
**Pull request by tnagorra**
- Add uuid in analytical_entries
- Add uuid in analytical_statements
- Rename `analytical_statement` to `analytical_statements`
- Replace `uuid` with `client_id`
cc @AdityaKhatri  @subinasr

### CODE REVIEW

**Comment by thenav56** [comment]
@tnagorra Both AnalyticalStatementEntry and AnalyticalStatement already have `cilent_id` field as they are inheriting the `UserResource` model. That 

https://github.com/the-deep/server/blob/9a84ece7e7883bf063282c366b810a9ec9823850/apps/user_resource/models.py#L5-L22

File: apps/analysis/models.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ class AnalyticalStatement(UserResource):
     )
     include_in_report = models.BooleanField(default=False)
     order = models.IntegerField()
+    uuid = models.CharField(max_length=255)
```
Comment (thenav56) [inline apps/analysis/models.py]: Not required. Let's use the already defined field `client_id`.

File: apps/analysis/models.py [hunk 2]
```diff
@@ -81,6 +82,7 @@ class AnalyticalStatementEntry(UserResource):
         on_delete=models.CASCADE
     )
     order = models.IntegerField()
+    uuid = models.CharField(max_length=255)
```
Comment (thenav56) [inline apps/analysis/models.py]: Not required. Let's use the already defined field `client_id`

File: apps/analysis/serializers.py [hunk 3]
```diff
@@ -21,7 +21,7 @@
 class AnlyticalEntriesSerializer(UserResourceSerializer):
     class Meta:
         model = AnalyticalStatementEntry
-        fields = ('id', 'order', 'entry')
+        fields = ('id', 'uuid', 'order', 'entry')
```
Comment (thenav56) [inline apps/analysis/serializers.py]: This should work.
```suggestion
        fields = ('id', 'client_id', 'order', 'entry')
```

**Review by thenav56 — CHANGES_REQUESTED** [review]
@tnagorra @k9845 Both AnalyticalStatementEntry and AnalyticalStatement already have `cilent_id` field as they are inheriting the `UserResource` model.

https://github.com/the-deep/server/blob/9a84ece7e7883bf063282c366b810a9ec9823850/apps/user_resource/models.py#L5-L22

File: apps/analysis/serializers.py (hunk 3 above)
Comment (k9845) [reply apps/analysis/serializers.py]: Done

**Review by thenav56 — APPROVED** [review]

**Review by annshress — APPROVED** [review]

**Pull request merged**